src/test/java/uk/gov/hmcts/reform/profilesync/helper/MockDataProvider.java

```diff
@@ -8,8 +8,8 @@
 import java.util.List;
 
 import uk.gov.hmcts.reform.profilesync.client.IdamClient;
-import uk.gov.hmcts.reform.profilesync.domain.GetUserProfileResponse;
 import uk.gov.hmcts.reform.profilesync.domain.UserProfile;
+import uk.gov.hmcts.reform.profilesync.domain.response.GetUserProfileResponse;
 
 public class MockDataProvider {
```

Comment:
if the variable  declaration is static final make  it in the capital letters .
example public static final String idamId  
change idamId to IDAMID

Reply:
Will make it how all constants are with underscores:  _IDAM_ID_

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-profile-sync/pull/181/commits/bc37603637e5fcbbc46bdd8621f47415d846e2c2